handlers/handlers.go

```diff
@@ -175,7 +174,7 @@ func versionsList(w http.ResponseWriter, req *http.Request, dc DatasetClient, re
 		return
 	}
 
-	p := mapper.CreateVersionsList(ctx, d, e, versions, cfg.EnableLoop11, cfg.EnableCookiesControl)
```

Comment:
Mmm bit too late to raise I guess, but, sending `ctx` and `req` as separate arguments is verbose. A request has a context on it. It is like doing the following:
`y = {x:1};`
`z = y.x;`
`something(y, z);`